modules/NuGetPackageVerifier/console/PackageIssueFactory.cs

```diff
@@ -39,7 +66,7 @@ public static PackageVerifierIssue AssemblyMissingServicingAttribute(string asse
                 PackageIssueLevel.Error);
         }
 
-        public static PackageVerifierIssue AssemblyInformationalVersionDoesNotMatchPackageVersion(string assemblyPath, NuGetVersion assemblyInformationalVersion, NuGetVersion packageVersion, string packageId)
+        public static PackageVerifierIssue AssemblyInformationalVersionDoesNotMatchPackageVersion(string assemblyPath, string assemblyInformationalVersion, NuGetVersion packageVersion, string packageId)
```

Comment:
Curious why we went from more to less specific?

Reply:
For when NuGetVersion.TryParse fails.